### DISCUSSION TITLE
Is it going to support GPU in the future

### DISCUSSION THREAD

**Discussion by Arky9782** [microsoft/WSL#6148] [Q&A] [answered]
So for now WSL2 doesn't have GPU, will it be added in the future? To avoid rendering on CPU

**Comment by ghost** [comment]
See this WSL 2 preview demo showcasing GPU Compute in a preview version of WSL 2 that's going to come in the future -> https://youtu.be/k3ZPzbQoV2M?t=2984

**Comment by ghost** [comment, marked as answer]
Checkout this [blog](https://devblogs.microsoft.com/commandline/gpu-compute-wsl-install-and-wsl-update-arrive-in-the-windows-insiders-fast-ring-for-the-windows-subsystem-for-linux/) by @craigloewen-msft as well
